docs/decisions/0002-text-editor-adr.md

```diff
@@ -0,0 +1,69 @@
+# Status
+
+Accepted
+
+# Context
+
+We seek to hoist the legacy text (HTML) text XBlock editor out of the monolith into a decoupled React application
+without reducing editor capabilities in the process.
+
+The legacy editor delegates the bulk of its operations to the [TinyMCE](https://www.tiny.cloud/docs/tinymce/6/) editor.
+Per the link, TinyMCE is "a rich-text editor that allows users to create formatted content within
+a user-friendly interface".
+
+# Decision set (each decision small)
+
+1. Rely on cross-editor XBlock network access support, per "Network Request Layer" ADR in this repo
+2. Continue using the tinyMCE editor
+3. Wrap the tinyMCE editor so as to be able to handle it as a vanilla React component
+4. Use a mix of configuration and tinyMCE API calls to customize the editor as needed 
+    * (e.g., offer image upload and image gallery selection)
+    
+# Consequences
+
+## Rely on editor-wide BlockStore networking decision
```

Comment:
nit: Just a note here to also remove any reference to blockstore

Reply:
I meant to say XBlock